stellar-core/debian/changelog

```diff
@@ -1,3 +1,7 @@
+stellar-core (_VERSION_-_BUILD_VERSION_) stable; urgency=medium
+
+  * added libcapstone-dev libfreetype6-dev libglfw3-dev libgtk2.0-dev Build-Depends
+
 stellar-core (_VERSION_-_BUILD_VERSION_) stable; urgency=medium
```

Comment:
This will cause duplicate version in the changelog as we update this file during build dynamically. It might cause problems but we can verify that.